app/src/main/java/com/ttbmp/cinehub/app/usecase/manageemployeesshift/request/GetHallListRequest.java

```diff
@@ -11,23 +11,23 @@
 
     public static final Request.Error MISSING_HALL = new Request.Error("sala non valida");
 
-    private CinemaDto cinema;
+    private Integer cinemaId;
```

Comment:
You should use the primitive type int for consistency reason if there isn't a strong reason to use a `java.lang.Integer` object